src/views/Pools/components/BountyCard.tsx

```diff
@@ -24,44 +36,68 @@ const BountyCard = () => {
   const { dollarCallBountyToDisplay, cakeCallBountyToDisplay, totalPendingCakeRewards } = useGetVaultBountyInfo(
     fastRefresh,
   )
+  const TooltipComponent = () => (
+    <Box>
+      <Box mb="16px">
+        {`${TranslateString(999, `This bounty is given as a reward for providing a service to other users.`)}`}
+      </Box>
+      <Box mb="16px">
+        {TranslateString(
+          999,
+          'Whenever you successfully claim the bounty, you’re also helping out by activating the Auto CAKE Pool’s compounding function for everyone.',
+        )}
+      </Box>
+      <Box style={{ fontWeight: 'bold' }}>
+        {TranslateString(999, `Auto-Compound Bounty: ${callFee / 100}% of all Auto CAKE pool users’ pending yield`)}
+      </Box>
+    </Box>
+  )
   const { callFee } = useGetVaultFees([FeeFunctions.callFee])
   const [onPresentBountyModal] = useModal(
     <BountyModal
       cakeCallBountyToDisplay={cakeCallBountyToDisplay}
       dollarCallBountyToDisplay={dollarCallBountyToDisplay}
       totalPendingCakeRewards={totalPendingCakeRewards}
       callFee={callFee}
+      TooltipComponent={TooltipComponent}
     />,
   )
 
+  const { targetRef, tooltip, tooltipVisible } = useTooltip(<TooltipComponent />, 'bottom-end')
+
   return (
-    <StyledCard>
-      <CardBody>
-        <Flex flexDirection="column">
-          <Flex alignItems="center" mb="12px">
-            <Text fontSize="16px" bold color="textSubtle" mr="4px">
-              {TranslateString(999, 'Auto CAKE Bounty')}
-            </Text>
-            <HelpIcon color="textSubtle" />
+    <>
+      {tooltipVisible && tooltip}
```

Comment:
Hyperactive diff. Probably happens throughout the PR when adding `{tooltipVisible && tooltip}` into the tree